fixtures/cover-01.pl

```diff
@@ -1,4 +1,9 @@
 
+BEGIN { my $b = 0 } # YES
+INIT  { my $c = 0 } # YES
+CHECK { my $d = 0 } # YES
+END   { my $e = 0 } # YES
+
```

Comment:
Probably need to improve the fixtures so that we say "YES" and say the expected phase.